src/sdl3/raw_window_handle.rs

```diff
@@ -57,105 +31,86 @@ unsafe impl HasRawWindowHandle for Window {
                 target_os = "netbsd",
                 target_os = "openbsd",
             ))]
-            SDL_SYSWM_WAYLAND => {
-                use self::raw_window_handle::WaylandWindowHandle;
+            "Linux" => {
```

Comment:
This should be something like
```
"Linux" | "OpenBSD" | "FreeBSD" | "NetBSD"
```

Reply:
Any idea where those would be defined?
I just took the strings from https://docs.rs/sdl3-sys/latest/sdl3_sys/platform/fn.SDL_GetPlatform.html

Reply:
> Any idea where those would be defined? I just took the strings from https://docs.rs/sdl3-sys/latest/sdl3_sys/platform/fn.SDL_GetPlatform.html

Here's the actual method's code:
https://github.com/libsdl-org/SDL/blob/3ebfdb04be8d543321524113951f817e0e2c56c4/src/SDL.c#L663

In addition, we can eliminate the call to `SDL_GetPlatform()` since the `cfg` attributes are already doing platform detection for us.

Reply:
Done, using #cfg